### PR TITLE
Minor change to cleanup things for SonarCloud - ensure ssl cert is validated

### PR DESCRIPTION
**Pull request by dwhite-sans**
Minor change to cleanup things for SonarCloud - ensure ssl cert is validated

### CODE REVIEW

File: srv/dshield/DShield.py [hunk 1]
```diff
@@ -19,7 +19,6 @@
 import hashlib
 import base64
 import requests
-import json
```
Comment (dwhite-sans) [inline srv/dshield/DShield.py:22]: This isn't being used

File: srv/dshield/DShield.py [hunk 2]
```diff
@@ -130,7 +129,7 @@ def ip62long(ip):
     # convert a long integer back to an IP address string
     @staticmethod
     def long2ip4(ip):
-        asciiip='127.0.0.1';
```
Comment (dwhite-sans) [inline srv/dshield/DShield.py:133]: not needed in python

File: srv/dshield/DShield.py [hunk 3]
```diff
@@ -217,9 +216,9 @@ def getopts(self,argv):
 
     def check_pid(self,pidfile):        
         """ Check For the existence of a unix pid. """
-        f=open(pidfile,'r')
-        pid=f.readline()
-        pid=pid.rstrip('\n')	
+        with open(pidfile, 'r') as file:
```
Comment (dwhite-sans) [inline srv/dshield/DShield.py:219]: using `with` syntax ensures the file is always closed, even on exception

**Comment by jullrich** [comment]
Thanks!

**Pull request merged**